pkg/components/components.go

```diff
@@ -271,15 +271,10 @@ func GetClusterRole() *rbacv1.ClusterRole {
 					"security.openshift.io",
 				},
 				Resources: []string{
-					"securitycontextconstraints",
-				},
-				ResourceNames: []string{
-					"privileged",
+					"*",
```

Comment:
This is what we are trying to avoid if possible. Could you just drop ResourceNames? I think that should be sufficient for our use-case